lib/cct/commands/openstack.rb

```diff
@@ -69,15 +74,29 @@ def create name, options={}
           OpenStruct.new(shell_parse(exec!(all_params).output))
         end
 
+        def add name, options={}
+          params.clear
+          yield params
+          all_params = ["add", name, "--format=shell"].concat(params.extract!(options))
+          OpenStruct.new(shell_parse(exec!(all_params).output))
+        end
+
+        def set name, options={}
+          params.clear
+          yield params
+          all_params = ["set", name].concat(params.extract!(options))
+          OpenStruct.new(shell_parse(exec!(all_params).output))
+        end
+
         def delete id_or_name
           params.clear
           exec!("delete", id_or_name)
         end
 
-        def list
+        def list *options
```

Comment:
@vmoravec That might be a bit hacky but I needed a simple way to add commandline options that you haven't provided.